driver/src/contracts/stablex_contract.rs

```diff
@@ -126,11 +126,11 @@ impl StableXContract for StableXContractImpl {
                     owners,
                     order_ids,
                     volumes,
-                    prices,
-                    token_ids_for_price,
+                    prices[1..],
```

Comment:
Should this logic maybe be part of `encode_prices_for_contract`? I was hoping to keep as much logic as possible outside of the `submit_solution` function as it's supposed to be dumb. In the other place it could be easily covered by a unit test.